### ISSUE TITLE
Hi! I can't run the demo

### ISSUE THREAD

**Issue by DurianBoomb** [wizgrav/clubber#15]
is it still can use?
I really like your work, it is very pity that your postprocessing can't use in 1.0.3, I wonder if aframe-clubber still can use in 1.0.3?
this is the error that demo throws
![image](https://user-images.githubusercontent.com/34919698/90965755-d82f7a80-e4c2-11ea-8422-ae453a105922.png)

**Comment by wizgrav** [comment]
I haven't touched the component in a very long time but it seems that it's a problem with loading the video. Let me see if I can cookup a simpler example

**Comment by wizgrav** [comment]
Yeap as I thought, the 360 video used as the backdrop throws 404, I should have actually copied it in the project not just reference the cdn. I found it and fixed it but the post proc is broken(I expected that). Right now post processing in VR is broken in three.js and I don't want anything to do with this, in fact I have switched to unity completely at this point and this is my recommendation. Regardless, the component should still be usable but I don't recommend using it as I will not be maintaining it anymore. Of course if anyone wants to take ownership, just pm me
